.pre-commit-hooks.yaml

```
@@ -14,24 +14,27 @@
   entry: hooks/snyk-iac.sh
   language: script
   pass_filenames: false
+  verbose: true
 
 - id: snyk-test
   name: Snyk Test
   description: Runs 'snyk test' command
   entry: hooks/snyk-test.sh
   language: script
   pass_filenames: false
+  verbose: true
 
 - id: snyk-code
   name: Snyk Code
   description: Runs 'snyk code test' command
   entry: hooks/snyk-code.sh
   language: script
   pass_filenames: false
+  verbose: true
 
 - id: snyk-log4shell
   name: Snyk log4shell
   description: Runs 'snyk log4shell' command
   entry: hooks/snyk-log4shell.sh
   language: script
-  pass_filenames: false
+  pass_filenames: false
```

```suggestion
  pass_filenames: false
  verbose: true
```
Let's add `verbose: true` here as well for consistency. I do not have strong opinion about having `true` here by default at this moment. Let's give it a shot and will see.